### DISCUSSION TITLE
influxdb GUI

### DISCUSSION THREAD

**Discussion by jdjohn555** [influxdata/influxdb#24362] [Show and tell]
Hi,

We recently added support for influxdb to qStudio.
qStudio is a free SQL GUI, for running SQL scripts, browsing tables and charting results.
https://www.timestored.com/qstudio/
We would like feedback from someone more experienced with influxdb.
If you are looking for a GUI, please give it a try and if you have any questions or feature requests
please raise an issue at:
https://github.com/timeseries/qstudioopen
![Screenshot (10)](https://github.com/influxdata/influxdb/assets/141037965/e27d7639-c951-4b63-97d5-bbfa5c2b1307)

**Comment by amberfan** [comment]
I tried but failed, I use qstudio to connect Influxdb 2.1.1
![image](https://github.com/influxdata/influxdb/assets/5319449/980c2416-a4f4-40bc-bcdb-efea6bb707c4)

![image](https://github.com/influxdata/influxdb/assets/5319449/79168171-d12d-4b47-96b4-9be72fd6e4bf)
